src/stories/select.stories.mdx

```diff
@@ -33,9 +33,24 @@ import gpioOptions from './assets/gpio-options'
       description: 'option to show the clear all button to deselect all otpions',
       table: { defaultValue: { summary: 'true' } }
     },
+    button: {
+      description: 'Whether a button exists at the bottom to conduct an action',
+      table: { defaultValue: { summary: 'false' } }
+    },
+    buttontext: {
+      description: 'The text on the button',
+      table: { defaultValue: { summary: 'ENTER' } }
+    },
+    buttonicon: {
+      description: 'The icon on the button, can use any from v-icon',
+      table: { defaultValue: { summary: '' } }
```

Comment:
might be nice to have an input for this for the Canvas view, just need to add `control: 'text'`. I'd say the same goes for buttontext

Reply:
added!